src/eureka/S3_data_reduction/plots_s3.py

```diff
@@ -1208,19 +1207,19 @@ def phot_2d_frame_diff(data, meta):
     - 2022-08-02 Sebastian Zieba
         Initial version
     """
-    for i in range(meta.nplots):
+    for i in range(meta.nplots-1):
```

Comment:
Ah yeah, so the issue was that if `meta.nplots` is set to None, then it gets set to the number of integrations which results in a crash when `i` gets to the last iteration since below we have `data.flux.values[i+1]` which goes beyond the end of the array. This was only meant as a temporary fix that I'd since forgotten about, and I'll tweak this so that it only subtracts 1 if `nplots==nints`